File: 02 (Even Odd 2)/p02.c
#include <stdio.h>
#include<string.h>

void solve() {
    // Your solution code goes here
}

int main() {
    #ifndef ONLINE_JUDGE
        //freopen("input.txt", "r", stdin);
        //freopen("output.txt", "w", stdout);
    #endif

    int tc;
    scanf("%d", &tc);

    for (int i = 0; i < tc; i++) {
        char s[20];
        int lastNumber;
        scanf("%s", s);

        lastNumber = s[strlen(s) - 1] - '0';

        if (lastNumber % 2 == 0) {
            printf("even\n");
        } else {
            printf("odd\n");
        }
    }
    return 0;
}

File: 02 (Even Odd 2)/p02.java
import java.util.Scanner;

public class Main {
    static void solve() {
        // Your solution code goes here
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int tc = scanner.nextInt();

        for (int i = 0; i < tc; i++) {
            String s = scanner.next();
            int lastNumber = Character.getNumericValue(s.charAt(s.length() - 1));

            if (lastNumber % 2 == 0) {
                System.out.println("even");
            } else {
                System.out.println("odd");
            }
        }
    }
}
